local/tests/test_rest_example.py

```diff
@@ -36,3 +39,31 @@ def test_help_all(self, base_endpoint):
             help_random_endpoint.json()["data"]["response"]
             == f"Unknown section '{random_arg}' in the --help-section option\n"
         )
+
+    def test_point(self, base_endpoint):
+        """This test is to test point endpoint with example data"""
+        point_endpoint = base_endpoint + "point"
+        directory_path = Path(os.getcwd())
+
+        config_path=(os.path.dirname(directory_path.parent.absolute())+"\\local\\rest_api_flint.example\\config\\point_example.json")
```

Comment:
Please change the URL endpoint to follow the Linux-based standards. Just change the format to `/` instead of `\\`.

Reply:
Also kindly use black formatter to format the code. Head over to your vscode settings and change the python formatter.

Reply:
@SanjaySinghRajpoot Thank you for pointing it out, will do it from now onwards!